runtime/doc/map.txt

```diff
@@ -1441,11 +1441,11 @@ The valid escape sequences are
 		expands to nothing.
 						*<mods>*
 	<mods>  The command modifiers, if specified. Otherwise, expands to
-		nothing. Supported modifiers are |:aboveleft|, |:belowright|,
-		|:botright|, |:browse|, |:confirm|, |:hide|, |:keepalt|,
-		|:keepjumps|, |:keepmarks|, |:keeppatterns|, |:leftabove|,
-		|:lockmarks|, |:noswapfile| |:rightbelow|, |:silent|, |:tab|,
-		|:topleft|, |:verbose|, and |:vertical|.
+                nothing. Supported modifiers are |:aboveleft|, |:belowright|,
```

Comment:
Tab should be used in the help.

Reply:
Ok.  Just pushed a new commit with tabs.

Just wondering though. Should help docs have `noet` in the modeline at the end of file? It already sets `ts=8`.